Examples/UtiliKit-iOSExample/Container/InteractiveTransition.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  InteractiveTransition.swift
+//  UtiliKit-iOS
+//
+//  Created by Will McGinty on 1/22/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```

Comment:
Can replace `Cocoapods` with `Bottle Rocket Studios`

Reply:
Can we go ahead and make that change in the project file settings as well?

Reply:
Changed all instances in project, but not sure what you mean @tylermilner

Reply:
@wmcginty If you click the Xcode project and then look in the Inspector Panel at the right, there's an "Organization" field. I'm guessing it's still set to "CocoaPods" from when we initially used their template to create the project.

![image](https://user-images.githubusercontent.com/1720304/61165732-c589fa00-a4e9-11e9-8551-af9da798274e.png)

Reply:
There it is. I feel like that was the one place in all of Xcode I didn't look. Fixed.